src/zoid/buttons/component.jsx

```diff
@@ -730,7 +736,7 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
                 [FPTI_KEY.CONTEXT_ID]: buttonSessionID,
                 [FPTI_KEY.CONTEXT_TYPE]: "button_session_id",
                 [FPTI_KEY.EVENT_NAME]: "message_click",
-                [FPTI_KEY.SELLER_ID]: merchantID?.join(","),
+                [FPTI_KEY.SELLER_ID]: merchantID,
```

Comment:
If we're including this above, do we also need it here?

Reply:
going to test this to see if that is the case

Reply:
Do we need this?